Clicker/ViewControllers/EditQuestionTypeViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  EditQuestionTypeViewController.swift
+//  Clicker
+//
+//  Created by Matthew Coufal on 9/18/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol EditQuestionTypeDelegate {
+    func didPick(questionType: QuestionType)
+}
+
+class EditQuestionTypeViewController: UIViewController {
+
+    var delegate: EditQuestionTypeDelegate?
+    var selectedQuestionType: QuestionType!
```

Comment:
put these in between `View vars` and `Constants` and add a mark for it called`// MARK: - Data vars`